### PR TITLE
Added canary switch

### LINKED ISSUE #25: Add canary-switch
Canaries will not work with limited resources on the vagrant box. Create a switch to turn it off and use it in example.

### PR DESCRIPTION
**Pull request by claesgill**
Closes #25 

- Added canary switch in both `presto.hcl` and `presto_standalone.hcl`
- Added `use_canary` variable
- Updated README

### CODE REVIEW

**Review by zhenik — APPROVED** [review]
LGTM

File: variables.tf [hunk 1]
```diff
@@ -106,6 +106,12 @@ variable "container_environment_variables" {
   default     = [""]
 }
 
+variable "use_canary" {
+  type = bool
+  description = "Uses canary deployment for Presto"
+  default = true
```
Comment (Neha-Sinha2305) [inline variables.tf:112]: *default* should be 'true' or 'false'?

Comment (claesgill) [reply variables.tf:112]: Before I added the switch it was enabled so I guess `true` makes sense. Or what do you think?

File: conf/nomad/presto_standalone.hcl [hunk 2]
```diff
@@ -10,10 +10,13 @@ job "${nomad_job_name}" {
     healthy_deadline  = "12m"
     progress_deadline = "15m"
     auto_revert       = true
-    auto_promote      = true
+  %{ if use_canary }
     canary            = 1
+    auto_promote      = true
+  %{ endif }
     stagger           = "30s"
   }
```
Comment (fredrikhgrelland) [inline conf/nomad/presto_standalone.hcl:18]:
```suggestion
  %{ if use_canary }
    canary            = 1
    auto_promote      = true
    auto_revert       = true
  %{ endif }
    stagger           = "30s"
  }
```

File: conf/nomad/presto_standalone.hcl [hunk 3]
```diff
@@ -10,10 +10,13 @@ job "${nomad_job_name}" {
     healthy_deadline  = "12m"
     progress_deadline = "15m"
     auto_revert       = true
```
Comment (fredrikhgrelland) [inline conf/nomad/presto_standalone.hcl:12]:
```suggestion
```

File: variables.tf (hunk 1 above)
Comment (fredrikhgrelland) [reply variables.tf:112]: I think false is good. canary requires double resources...

**Review by fredrikhgrelland — COMMENTED** [review]
We need auto_revert also

File: conf/nomad/presto_standalone.hcl (hunk 2 above)
Comment (claesgill) [reply conf/nomad/presto_standalone.hcl:18]: Ok, it didn't give any errors so didn't think of it! I'll commit your suggestion.

**Review by dangernil — APPROVED** [review]

**Pull request merged**